torchvision/transforms/transforms.py

```diff
@@ -2036,19 +2030,19 @@ def forward(self, batch: Tensor, target: Tensor) -> Tuple[Tensor, Tensor]:
         batch_flipped = batch.flip(0)
         target_flipped = target.flip(0)
 
-        if self._mixup_dist is None:
+        if self.mixup_alpha <= 0.0:
             use_mixup = False
         else:
-            use_mixup = self._cutmix_dist is None or torch.rand(1).item() >= self.cutmix_p
+            use_mixup = self.cutmix_alpha <= 0.0 or torch.rand(1).item() >= self.cutmix_p
 
         if use_mixup:
             # Implemented as on mixup paper, page 3.
-            lambda_param = self._mixup_dist.sample().item()
+            lambda_param = float(torch._sample_dirichlet(torch.tensor([self.mixup_alpha, self.mixup_alpha]))[0])
```

Comment:
The above is the JIT-friendly version of the following: 
```python
torch.distributions.Beta(self.mixup_alpha, self.mixup_alpha).sample()
```

Dirichlet is the generalization of Beta for multiple variables.